// angular-cd-timer/esm2020/lib/angular-cd-timer.module.mjs
import { NgModule } from '@angular/core';
import { CdTimerComponent } from './angular-cd-timer.component';
import * as i0 from "@angular/core";
export class CdTimerModule {
}
CdTimerModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: CdTimerModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
CdTimerModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: CdTimerModule, declarations: [CdTimerComponent], exports: [CdTimerComponent] });
CdTimerModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: CdTimerModule, imports: [[]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: CdTimerModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [CdTimerComponent],
                    imports: [],
                    exports: [CdTimerComponent]
                }]
        }] });
